src/main/java6/net/finmath/marketdata/model/curves/ForwardCurveInterface.java

```diff
@@ -15,16 +15,23 @@
 public interface ForwardCurveInterface extends CurveInterface {
 
 	/**
+	 * Returns the forward for the corresponding fixing time without a given model. Note that this does not work for all interpolation methods as some of them require a model
+	 * @param fixingTime The fixing time of the index.
+	 * @return The forward
+	 */
+	double getForward(double fixingTime);
```

Comment:
Yes, I think your concern is very much legit. Can you undo this modification when accepting the pull-request or should I undo it via a commit?